test/test_all.py

```diff
@@ -99,6 +102,8 @@ def __init__(self, script, jitter ,*args, **kwargs):
         if jitter in blacklist.get(script, []):
             continue
         tags = [TAGS[jitter]] if jitter in TAGS else []
+        if is_win and script.endswith("mn_div.py"):
```

Comment:
I see the `mn_div` test is bypassed on Windows, is there a specific reason?

Reply:
Yes!

I believe I'd need libLLVM for this test to work and unfortunately libLLVM isn't a binary that is shipped in the clang/llvm Windows package they ship.

So I'd need to probably compile it myself in the CI which I haven't tried out yet. We could open an issue for it and assigning it to me if you want?